corehq/apps/style/management/commands/fix_less_imports_collectstatic.py

```diff
@@ -23,11 +21,25 @@ def handle(self, *args, **options):
         for less_file in collectedless:
             with codecs.open(less_file, 'r', 'utf-8') as fd:
                 content = fd.read()
-                if NON_COLLECTED_PATH in content:
-                    print "[FIXING] bad less import in %s" % less_file
-                    content = content.replace(NON_COLLECTED_PATH, COLLECTED_PATH)
-                else:
-                    content = None
+                if content is not None:
+                    B3_REGEX = r"@\{b3-import-[a-z]+-[a-z]+\}"
+                    p = re.search(B3_REGEX, content)
+                    if p is not None:
+                        path_def = p.group(0)
+                        path_file = path_def.split('-')[-2]
```

Comment:
maybe just a comment here to say what this is doing? it looks like you're determining whether it's a mixin or variables. or just add grouping to the regex